File: Som.PerformanceMeasurement/TestResult.cs
using System;
using System.Collections.Generic;

namespace PerformanceMeasurement
{
    [Serializable]
    public class TestResult
    {
        public TestResult(){}
        public TestResult(int iterations, string gridSize, int inputSpaceDimentions)
        {
            Iterations = iterations;
            GridSize = gridSize;
            InputSpaceDimentions = inputSpaceDimentions;
            TimeStat = new List<AlgorithmTime>();
        }

        public int Iterations { get; set; }
        public string GridSize { get; set; }
        public int InputSpaceDimentions { get; set; }
        public List<AlgorithmTime> TimeStat { get; set; }
    }

    [Serializable]
    public class AlgorithmTime
    {
        public AlgorithmTime() { }
        public AlgorithmTime(string algoName, double aveTime)
        {
            AlgoName = algoName;
            AverageTime = aveTime;
        }

        public string AlgoName { get; set; }
        public double AverageTime { get; set; }
    }
}

File: JavaSom/TESTER/TestResult.java
package TESTER;

import sunw.io.Serializable;

public class TestResult implements Serializable
	{
		public TestResult()
		{
		}
		public TestResult(int iterations, String gridSize, int inputSpaceDimentions)
		{
			setIterations(iterations);
			setGridSize(gridSize);
			setInputSpaceDimentions(inputSpaceDimentions);
			setTimeStat(new java.util.ArrayList<AlgorithmTime>());
		}

		private int privateIterations;
		public final int getIterations()
		{
			return privateIterations;
		}
		public final void setIterations(int value)
		{
			privateIterations = value;
		}
		private String privateGridSize;
		public final String getGridSize()
		{
			return privateGridSize;
		}
		public final void setGridSize(String value)
		{
			privateGridSize = value;
		}
		private int privateInputSpaceDimentions;
		public final int getInputSpaceDimentions()
		{
			return privateInputSpaceDimentions;
		}
		public final void setInputSpaceDimentions(int value)
		{
			privateInputSpaceDimentions = value;
		}
		private java.util.ArrayList<AlgorithmTime> privateTimeStat;
		public final java.util.ArrayList<AlgorithmTime> getTimeStat()
		{
			return privateTimeStat;
		}
		public final void setTimeStat(java.util.ArrayList<AlgorithmTime> value)
		{
			privateTimeStat = value;
		}
	}